element-connector/src/test/java/org/eclipse/californium/elements/tcp/TlsConnectorTest.java

```diff
@@ -35,6 +35,8 @@
 
 public class TlsConnectorTest {
 
+	private static final int NUMBER_OF_THREADS = 1;
```

Comment:
About the formatting: 
sorry I'm a little lost:

```
	@Override public void destroy() {
		for (Layer layer : layers)
			layer.destroy();
	}

		@Override public void receiveRequest(Exchange exchange, Request request) {
			// if there is no BlockwiseLayer we still have to set it
			if (exchange.getRequest() == null)
				exchange.setRequest(request);
			if (hasDeliverer()) {
				deliverer.deliverRequest(exchange);
			} else {
				LOGGER.severe("Top of CoAP stack has no deliverer to deliver request");
			}
		}
```

So, is it OK to correct only my changes, or is more required?